### PR TITLE
Reorder options: Move general options to the left side and layout and text options to the right side of options menu

### PR DESCRIPTION
**Pull request by tobias-klein**
@zhuiks
@augustin-colesnic 
I would prefer this order of the optoin groups in the options menu.
Note that there were also some changes within the options groups compared to 1.0.0.
Please have a look and give feedback.

Please approve via the GitHub review functionality if you are OK with this change.

**NEW** (1.1.0)

image

**OLD** (1.0.0)

image

### CODE REVIEW

**Comment by zhuiks** [comment]
@tobias-klein can we put "Night mode" as a first one in **Display** section. I think that's the standard option that users would look for

File: html/display_options_menu.html [hunk 1]
```diff
@@ -26,11 +34,14 @@
     <option value="open-chapters-all-books" label="bible-browser.open-chapters-all-books"></option>
   </select>
 
+  <config-option id="openVerseListsInNewTabOption" settingsKey="openVerseListsInNewTab" label="bible-browser.open-verse-lists-in-new-tab" checkedByDefault="false"></config-option>
+  <config-option id="adjustTagsNotesTextSizeOption" settingsKey="adjustTagsNotesTagSize" label="bible-browser.adjust-font-size-tags-notes" checkedByDefault="false"></config-option>
+  <config-option id="fixNotesHeightOption" settingsKey="fixNotesHeight" label="bible-browser.limit-notes-display-size" checkedByDefault="false"></config-option>
   <config-option id="useNightModeOption" settingsKey="useNightMode" label="bible-browser.use-night-mode" checkedByDefault="false" autoLoad="false"></config-option>
   <config-option id="keepScreenAwakeOption" settingsKey="keepScreenAwake" label="bible-browser.keep-screen-on" checkedByDefault="false"></config-option>
```
Comment (zhuiks) [inline html/display_options_menu.html:41]: How about moving these two option as first ones in **Display** section (insert after line 28). These are standard options and users would look for them.

**Review by zhuiks — CHANGES_REQUESTED** [review]
@tobias-klein This is good rearrangement! If we will move this to a right one column sidebar as discussed in https://github.com/ezra-bible-app/ezra-bible-app/discussions/299#discussioncomment-777004 it would be nice to keep the same new order.

File: html/display_options_menu.html (hunk 1 above)
Comment (tobias-klein) [reply html/display_options_menu.html]: Yes, makes sense. I just pushed that change.

**Comment by tobias-klein** [comment]
@zhuiks I implemented the change you requested. Please approve the PR.

**Comment by augustin-colesnic** [comment]
Hi @tobias-klein , 

I fails to see night mode on the settings.

Screenshot 2021-06-21 at 11 17 11
Screenshot 2021-06-21 at 11 25 08

And @zhuiks  I found uncomfortable the timeout that makes the settings panel disappear after I check an option...  Old behaviour was better.

**Review by zhuiks — APPROVED** [review]
Thanks!

**Comment by tobias-klein** [comment]
@tobias-klein On macOS the night mode is controlled via system settings only. When you change it in the system settings the app will respond to that change right away.

**Pull request merged**